addons/project/static/src/js/tours/project.js

```diff
@@ -167,14 +167,6 @@ registry.category("web_tour.tours").add('project_tour', {
     isActive: ["auto"],
 },
 {
-    isActive: ["auto"],
-    trigger: ".o_field_widget[name='user_ids'] input",
-    content: _t("Assign a responsible to your task"),
-    position: "right",
-    run: "edit Mitch",
-},
-{
-    isActive: ["manual"],
```

Comment:
these changes should be in the commit containing project changes. 🙂

Reply:
Hey @xavierbol This is the final iteration of the solution possible as click is not actuating the dropdown by typing something and clear it we are able to actuate the dropdown and all the options will be available.PIPU also provided a solution 

```{
    isActive: ["auto"],
    trigger: ".o_field_widget[name='user_ids'] input",
    content: _t("Assign a responsible to your task"),
    position: "right",
    run: "edit Mitch",
},
{
    isActive: ["auto", "desktop"],
    trigger: ".o_field_widget[name='user_ids'] .o-autocomplete--dropdown-menu li:contains(Mitchell Admin)",
    run: "click",
},
{
    isActive: ["manual"],
    trigger: ".o_field_widget[name='user_ids'] input",
    content: _t("Assign a responsible to your task"),
    position: "right",
    run: "click",
},
{
    isActive: ["manual", "desktop"],
    trigger: ".ui-autocomplete > li > a:not(:has(i.fa))",
    run: "click",
},
```

Both solutions are identical. It is only suspicious that ```run: click``` works in auto testing tour but not in headless mode.

Also PIPU confirms that headless behaviour is not same as onboarding quoted as 
```with start_tour, the behavior is not the same as in onbaording ... to run test in onboarding, you must start localhost, then debug mode and then click on start tour ( manually )```